src/service/ampdoc-impl.js

```diff
@@ -183,6 +193,16 @@ export class AmpDoc {
   constructor(win) {
     /** @public @const {!Window} */
     this.win = win;
+
+    /** @private {boolean} */
+    this.isRenderStarted_ = false;
```

Comment:
Do you mean you want to apply the same signals instrument to ampdocs as well? That's probably a good idea. Will do.